keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -326,7 +326,17 @@ func checkKubernetesVersion(serverVersion *version.Info) error {
 //installKubeEdge downloads the provided version of KubeEdge.
 //Untar's in the specified location /etc/kubeedge/ and then copies
 //the binary to excecutables' path (eg: /usr/local/bin)
-func installKubeEdge(componentType types.ComponentType, arch string, version semver.Version) error {
+func installKubeEdge(options types.InstallOptions, arch string, version semver.Version) error {
+	//create the storage path of the kubeedge installation packages
+	if options.TarballPath == "" {
+		options.TarballPath = KubeEdgePath
+	} else {
+		err := os.MkdirAll(options.TarballPath, os.ModePerm)
+		if err != nil {
+			return fmt.Errorf("not able to create %s folder path", options.TarballPath)
+		}
+	}
+
 	err := os.MkdirAll(KubeEdgePath, os.ModePerm)
```

Comment:
Does TarballPath cover all files we need? If so, why need `KubeEdgePath`? We can set it as TarballPath default value.

Reply:
In the existing implementation, we just put some larger installation packages under the TarballPath, while the KubeEdgePath  will store some relatively small content, such as logs, certificates, etc.

Reply:
It may be confused for users, need to comment in help info.

Also, it would be better to copy to `KubeEdgePath` first then unzip.